### DISCUSSION TITLE
Folder name grouping or scroll in left pane

### DISCUSSION THREAD

**Discussion by abhisheksingh0196** [zed-industries/zed#6644] [Feature Requests]
Hi everyone, this is my first time posting a question in Github discussion. Let me know if I should be adding more info

I've seen VS Code grouping folder name in case the folder only contains other folder. Zed just expands this with indent. And there is no scroll on the left pane, so if file/folder names go out of view I have to expand the left pane size.

Is there a solution for this through some settings? 

<img width="266" alt="Screenshot 2023-12-07 at 2 20 52 PM" src="https://github.com/zed-industries/community/assets/97225179/4dea262d-1842-47df-a7ff-ecbcbbf964fc">

**Comment by JosephTLyons** [comment]
This is a missing feature. The only workaround at the moment is expanding the dock to reveal more. Its not ideal at all - we will fix this after we finish up the port to our GPUI2.
